commons-bio/src/main/kotlin/ebi/ac/uk/util/EitherExtensions.kt

```
@@ -0,0 +1,12 @@
+package ebi.ac.uk.util
+
+import arrow.core.Either
+import arrow.core.getOrElse
+
+fun <A, B> MutableList<Either<A, B>>.addLeft(element: A) = add(Either.left(element))
+
+fun <A, B> MutableList<Either<A, B>>.addRight(element: B) = add(Either.right(element))
+
+fun <A, B> Either<A, B>.getLeft(): A = swap().getOrElse { throw Exception("It's not a left element") }
```

I do not like `getLeft` and `getRight` here, there should be used only in test code not in real code hiding a possible exception.